webapp/.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```

Comment:
This file is used by `asdf`, runtime manager that I use. You don't need to use it.

Reply:
Thanks, I'm definitely gonna explore ASDF soon too.